assets/readme.md

```diff
@@ -1,15 +1,59 @@
+
+## Features
+
+The SQL connector supports to `create users`, `add traits` and `update traits`.
+
+You can bring in new users or update existing user profiles from your SQL databases to create segments, transform and enrich customer data and send them to other services with our other Connectors.
+
+Hull SQL Connector allows you to synchronize data on a given schedule, define how data is treated in terms of conflicts and run incremental queries.
+
+## Synchronize data on a scheduled basis
+
+**Enable Sync** to run your query on a given time interval to keep your data automatically up-to-date:
+![Synchronize Data](./docs/syncschedule01.png)
+
+By checking the box your query runs automatically every 3 hours. To increase performance we recommend to use incremental queries.
+
+Note: You can always manually run your query from the dashboard by clicking on the button “Import everything”.
+
+## Define behavior in case of a data conflict
+
+A user profile in Hull is usually composed of data from a multitude of sources, so it is likely that you have already stored a value for a given trait and your SQL data source returns a different value. Let’s say you have already stored the phone number 123-444-6666 in the user profile for Brad Smith but your query returns the number 456-233-8899. This represents a typical data conflict and you can decide how the SQL connector shall resolve this conflict: either keep the number or overwrite it. By selecting **Always overwrite data in Hull with values from SQL** in the connector configuration section, you make the decision that your SQL data source is the leading system for the particular traits it returns:
+![Handle Data Conflicts](./docs/dataconflict01.png)
+
+## Query data that has changed in a given time period (Incremental Queries)
+
+If the tables in your SQL database hold large sets of data, you might want to query only data that has changed within the last couple of days - this is what we call incremental queries. The advantage of incremental queries is that you return a smaller subset of data that can be processed faster which improves the overall performance. You can write an incremental query by using the placeholder `:import_start_date`  in your query string. The SQL connector will automatically replace this at runtime with a proper datetime value that represents the current point of time x days ago. Here is an example of an incremental query:
+![Incremental Queries Dashboard](./docs/incrementalqueries01.png)
+You can define the number of days on the tab “Settings“ in the section “Connector Configuration”:
+![Incremental Queries Settings](./docs/incrementalqueries02.png)
+Please enter the number of days as integer or whole number. Fractional days are not supported.
+
+## Supported SQL Server Types
+
+- Redshift
+- PostgreSQL
+- MySQL
+- Microsoft SQL
+
+---
+
+If you have any questions or want to suggest improvements to our documentation, please [let us know](https://hull.io).
```

Comment:
Agree w/sven here support should be standardized, and it already is somehow: when the readme is served in the Dashboard or via the /readme link (https://hull-sql.herokuapp.com/readme) intercom is added along with a footer containing various links (https://github.com/hull/hull-admin/blob/develop/server/views/readme/footer.ejs)